task02/task02/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task02
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите длину стороны правильного икосаэдра");
+            double a = double.Parse(Console.ReadLine());
+
+            double x = Math.Pow(a, 2);
+            double c = 3;
+            double y = Math.Sqrt(c);
+            double s = 5 * x * y;
+            double k = Math.Pow(a, 3);
+            double b = 5;
+            double z = Math.Sqrt(b);
+            double v = (5 * k * (3 + z)) / 12;
```

Comment:
Слишком много лишних промежуточных локальных переменных. Они только затрудняют чтение кода, а их значения используются однократно, то есть храни ть их отдельно в памяти смысла нет. Луше каждое вычисление написать в одну формулу. Квадрат числа проще записать через умножение
```suggestion
            double s = 5 * Math.Sqrt(3) * a * a;
            double v = 5 *  (3 + Math.Sqrt(5)) * a * a * a / 12;
```